pkg/scheduler/framework/plugins/examples/multipoint/multipoint.go

```diff
@@ -84,8 +83,3 @@ func (mc CommunicatingPlugin) PreBind(ctx context.Context, state *framework.Cycl
 	}
 	return nil
 }
-
-// New initializes a new plugin and returns it.
-func New(_ *runtime.Unknown, _ framework.Handle) (framework.Plugin, error) {
```

Comment:
This is an example, so we should keep it.

Reply:
Done

Reply:
revert all change in `pkg/scheduler/framework/plugins/examples`